app/javascript/elm/src/Popup.elm

```diff
@@ -16,6 +16,12 @@ type PopupPart
     | OtherPart
 
 
+type PopupStatus
+    = ParameterPopupShown
+    | SensorPopupShown
+    | PopupHidden
```

Comment:
this seems to similar to `type Popup`. If indeed parameter and sensor popup are different then I would just have one type:
```
type Popup
    = ParameterList someArgs
    | SensorList someDifferentArgs
    | None
```
if they are the same then what we already have on line 9 should be enough:
```
type Popup
    = SomeList someArgs
    | None
```

Reply:
the former sounds like the way to go.